components/consents/api/pom.xml

```diff
@@ -29,7 +29,8 @@
   <name>PhenoTips - Consents - Java APIs</name>
 
   <properties>
-    <coverage.instructionRatio>0.80</coverage.instructionRatio>
+    <checkstyle.suppressions.location>${basedir}/src/checkstyle/checkstyle-suppressions.xml</checkstyle.suppressions.location>
```

Comment:
I'm getting a build fail due to checkstyle violation (it can't find checkstyle-suppressions.xml).